### ISSUE TITLE
CancellationException

### ISSUE THREAD

**Issue by Nacoma** [gtache/intellij-lsp#77]
This one happened upon opening a vendor/library source file in v1.5.3

```
2019-03-07 09:57:29,515 [ 150176]  ERROR - plication.impl.ApplicationImpl - PhpStorm 2018.3.5  Build #PS-183.5912.26 
2019-03-07 09:57:29,515 [ 150176]  ERROR - plication.impl.ApplicationImpl - JDK: 1.8.0_152-release; VM: OpenJDK 64-Bit Server VM; Vendor: JetBrains s.r.o 
2019-03-07 09:57:29,515 [ 150176]  ERROR - plication.impl.ApplicationImpl - OS: Linux 
2019-03-07 09:57:29,512 [ 150173]  ERROR - plication.impl.ApplicationImpl - null 
java.util.concurrent.CancellationException
	at java.util.concurrent.CompletableFuture.cancel(CompletableFuture.java:2263)
	at com.github.gtache.lsp.client.languageserver.wrapper.LanguageServerWrapperImpl.stop(LanguageServerWrapperImpl.scala:244)
	at com.github.gtache.lsp.client.languageserver.wrapper.LanguageServerWrapperImpl.disconnect(LanguageServerWrapperImpl.scala:239)
	at com.github.gtache.lsp.PluginMain$.$anonfun$editorClosed$2(PluginMain.scala:278)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:314)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by gtache** [comment]
I guess you closed it rapidly after? Given that the stacktrace contains editorClosed.     
Anyway, that should be fixed in 1.6.0. If I'm not releasing it soon I will publish a 1.5.4 version.

**Comment by Nacoma** [comment]
>I guess you closed it rapidly after? Given that the stacktrace contains editorClosed.      ...

@gtache yes, I believe that I had opened it by mistake.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
